package.json

```diff
@@ -13,18 +13,28 @@
     "LICENSE"
   ],
   "devDependencies": {
+    "@babel/core": "^7.4.4",
```

Comment:
I'm not sure if we already agreed to use `^` on external dependencies... or something changed?